packages/devtools-launchpad/package.json

```diff
@@ -102,6 +100,7 @@
     "@kadira/storybook": "^2.35.3",
     "ipaddr": "^0.0.9",
     "jest": "^19.0.2",
+    "svg-inline-loader": "^0.8.0",
```

Comment:
if we have svg loader, should we use that? If not we can remove this:

this is how we use it
https://github.com/devtools-html/debugger.html/blob/master/assets/images/Svg.js#L1

Reply:
I use it in webpack.config to load svg